python/bosdyn-mock/src/bosdyn/mock/state.py

```
@@ -0,0 +1,382 @@
+"""In-memory state shared across the mock services."""
```

Pylint (4.0.2)

[error] Unrecognized option found: suggestion-mode, no-space-check

(E0015)